File: migration/pset1/cash/cash.c
#include<cs50.h>
#include<stdio.h>
#include<math.h>

int main(void)
{
    float dollars;
    
    // inputs change owed from user
    do
    {
        dollars = get_float("Enter change owed\n");
    }
    while (dollars < 0);
    
    int cents = round(dollars * 100);
    int coins = 0;
    
    // counts minimum coins required
    
    while (cents != 0)
    {
        
        if (cents >= 25)
        {
            cents = cents - 25;
            coins++;
        }
        else if (cents >= 10)
        {
            cents = cents - 10;
            coins++;
        }
        else if (cents >= 5)
        {
            cents = cents - 5;
            coins++;
        }
        else
        {
            cents = cents - 1;
            coins++;
        }
    }
    printf("Total coins to be given: %i\n", coins);
}

File: sentimental-cash/cash.py
# TODO

from cs50 import get_float

while True:
    cents = get_float("Change owed: ")
    if cents > 0:
        break

count = 0
while cents > 0:
    if cents >= 0.25:
        cents -= 0.25
        count += 1
    elif cents >= 0.10:
        cents -= 0.10
        count += 1
    elif cents >= 0.05:
        cents -= 0.05
        count += 1
    else:
        cents -= 0.01
        count += 1

print(count)